responder/ext/__init__.py

```diff
@@ -1 +1,6 @@
-from .graphql import GraphQLView
+try:
+    from .graphql import GraphQLView
+except ModuleNotFoundError:
+    # ignore graphql-server dependency errors as the extension is loaded even
+    # it is not used. keep extension import for compat with precedent releases.
+    pass
```

Comment:
Can you remove this import from the init file and add a try catch around the external dependency with a nice message e.g.
```python
try:
    from graphql_server import A, B, C
except ModuleNotFoundError:
     raise ModuleNotFoundError("responder['graphql'] must be installed in order to use `GraphQLView`") 
```

PS: same for the open api schema